node/metrics/Cargo.toml

```diff
@@ -8,10 +8,18 @@ description = "Subsystem traits and message definitions"
 [dependencies]
 futures = "0.3.17"
 futures-timer = "3.0.2"
+tracing = "0.1.29"
 
 metered-channel = { path = "../metered-channel" }
 
 substrate-prometheus-endpoint = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-tracing = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-cli = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-service = { git = "https://github.com/paritytech/substrate", branch = "master" }
+codec = { package = "parity-scale-codec", version = "2.2.0" }
+sp-tracing = { git = "https://github.com/paritytech/substrate", branch = "master" }
+primitives = { package = "polkadot-primitives", path = "../../primitives/" }
+bs58 = { version = "0.4.0", features = ["alloc"] }
```

Comment:
Do we really all of these deps, especially `sc-service` and `sc-cli` are particularly heavy.

Reply:
`sc-service`  shouldn;t be there, sc-cli is needed for `FnOnce(&mut sc_cli::LoggerBuilder, &sc_service::Configuration) -> ()`

Reply:
Yikes. Well, it is what it is for now. Please add a comment to the why this is needed.